src/taxinyc/data_processor.py

```
@@ -0,0 +1,87 @@
+
+class DataProcessor:
+    def __init__(self, sparksession, table, config):
+        self.df = self.load_data(sparksession, table)
+        self.config = config
+        self.X = None
+        self.y = None
+        self.preprocessor = None
+    
+    def load_data(self, sparksession, table: str) -> pd.DataFrame:
+        """Load data from Spark table into pandas DataFrame.
+        
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)